ql/src/java/org/apache/hadoop/hive/ql/exec/SerializationUtilities.java

```diff
@@ -573,31 +665,23 @@ public PartitionDesc read(Kryo kryo, Input input, Class<? extends PartitionDesc>
    * @param out  The stream to write to.
    */
   public static void serializePlan(Object plan, OutputStream out) {
-    serializePlan(plan, out, false);
-  }
-
-  public static void serializePlan(Kryo kryo, Object plan, OutputStream out) {
-    serializePlan(kryo, plan, out, false);
+    serializePlan(plan, out, null);
   }
 
-  private static void serializePlan(Object plan, OutputStream out, boolean cloningPlan) {
-    Kryo kryo = borrowKryo();
+  private static void serializePlan(Object plan, OutputStream out, Configuration configuration) {
```

Comment:
Can you add simple test case for ser/deser with these props in TestPlan.java?

Reply:
yeah, I've just created a new test class, TestSerializationUtilities